cmds/wiki/user.js

```diff
@@ -321,19 +321,22 @@ function gamepedia_user(lang, msg, namespace, username, wiki, querystring, fragm
 			var isBlocked = false;
 			var blockedtimestamp = ( queryuser.blockedtimestamp ? dateformat.format(new Date(queryuser.blockedtimestamp)) : 'Invalid Date' );
 			var blockexpiry = queryuser.blockexpiry;
+			var isPermabanned = queryuser.isPermabanned;
```

Comment:
`queryuser.isPermabanned` is always `undefined`, why is it used here?

Reply:
That's meant to be a boolean to indicate if the user's permanently banned which then uses an alternative message.

Reply:
But `queryuser` has no `isPermabanned` property so it shouldn't be used here. It's literally the same as setting it to `undefined` but in a very ugly way.

Reply:
Changed it into a bool, how does it sound?